FirebaseStorage/Tests/Unit/StorageTests.swift

```diff
@@ -57,14 +59,14 @@ class StorageTests: XCTestCase {
     XCTAssertEqual("gs://foo-bar.appspot.com/", storage2.reference().description)
   }
 
-  func testInitWithWrongScheme() throws {
+  func SKIPtestInitWithWrongScheme() throws {
```

Comment:
Now generating a fatalError instead of throwing an Objective C exception. We should add tests back if we add an API for throwing Swift exceptions.